apps/organizations/views.py

```diff
@@ -67,25 +65,27 @@ def setup(self, request, *args, **kwargs):
         self.data['invitation'] = self.invitation
 
     def get(self, request, *args, **kwargs):
+        is_mobile = request.user_agent.is_mobile or request.user_agent.is_tablet
+        mobile_url_format = htk_setting(
```

Comment:
dislike bc this makes the LHS/RHS names mismatch later

Maybe just go with boring (but consistent, symmetrical):

`mobile_invitation_response_url_format`